lib/libmbedtls/mbedtls/library/bignum.c

```diff
@@ -100,7 +116,10 @@ void mbedtls_mpi_free( mbedtls_mpi *X )
     if( X->p != NULL )
     {
         mbedtls_mpi_zeroize( X->p, X->n );
-        mbedtls_free( X->p );
+        if (X->use_mempool)
```

Comment:
nitpicking on mbedtls coding style, i saw that there should be space around parentheses whatever the parenthesis is used for.

Reply:
I'll fix